
pmax: 7600.0
Traceback (most recent call last):
  File "/home/pedro/PINN/jaxpi/examples/geomIII2/main.py", line 39, in <module>
    app.run(main)
  File "/usr/local/lib/python3.11/dist-packages/absl/app.py", line 308, in run
    _run_main(main, args)
  File "/usr/local/lib/python3.11/dist-packages/absl/app.py", line 254, in _run_main
    sys.exit(main(argv))
             ^^^^^^^^^^
  File "/home/pedro/PINN/jaxpi/examples/geomIII2/main.py", line 31, in main
    train.train_and_evaluate(FLAGS.config, FLAGS.workdir)
  File "/home/pedro/PINN/jaxpi/examples/geomIII2/train.py", line 53, in train_and_evaluate
    print(dsadas)
          ^^^^^^
NameError: name 'dsadas' is not defined